atlp-rwanda/rca-barefoot-team4-fe
# feat: added the user exploration home page

**What does this PR do?**
This PR adds a new user exploration home page that allows users to search for flights, hotels, and cars, and pass the departure dates. However, the API is not integrated yet.

**Description of Task to be completed?**
The task completed by this PR is to create a new user exploration home page that allows users to search for flights, hotels, and cars and pass the departure dates.

**How should this be manually tested?**
To test this feature manually, navigate to the new user exploration home page and attempt to search for flights, hotels, and cars. Ensure that the departure date can be passed and that the appropriate results are displayed.

For now this is the root route: test by accessing http://localhost:3000/

**Any background context you want to provide?**
This feature was developed as part of an effort to improve the user experience of the travel booking platform.

**What are the relevant pivotal tracker/Trello stories?**
There are no relevant pivotal tracker/Trello stories associated with this PR at this time.

**Screenshots (if appropriate)**
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/54812621/235971021-4ef61e30-413e-478b-875f-67e955cf3463.png">



**Questions:**
There are no questions included in the commit message.
